packages/@okta/courage-for-signin-widget/webpack.config.js

```diff
@@ -36,13 +26,10 @@ const webpackConfig = {
     // node_modules at root directory.
     path: path.resolve(__dirname, '../'),
     filename: `${DIST_FILE_NAME}.js`,
-    libraryTarget: 'umd',
-    library: DIST_FILE_NAME
+    libraryTarget: 'commonjs2'
```

Comment:
Are we ok changing the target to `commonjs2`? This seems like we're introducing a breaking change.

Reply:
I think it's fine because this file is only being used by signin widget internally. `umd` target is for broader audience.

Reply:
Awesome - just wanted to be sure.